includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -484,17 +499,21 @@ private function getSitenameData() {
 	}
 
 	private function getVerticalData() {
-		$wikiFactoryInstance = WikiFactoryHub::getInstance();
-		$verticalData = $wikiFactoryInstance->getWikiVertical( $this->wikiId );
-
-		/**
-		 * We don't want to show vertical 'Other' instead we show vertical 'Lifestyle'
-		 * This is Comscore requirement
-		 */
-		if ( $verticalData['id'] == WikiFactoryHub::VERTICAL_ID_OTHER ) {
-			$verticalMessageKey = $wikiFactoryInstance->getAllVerticals()[WikiFactoryHub::VERTICAL_ID_LIFESTYLE]['short'];
+		if ( $this->product === self::FANDOM_PRODUCT ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)